File: src/data/snippets/functions/snippet.java
// Function to calculate the area of a circle
public static double calculateCircleArea(double radius) {
    // Area formula using Math.pow()
    return Math.PI * Math.pow(radius, 2);  
}

public static void main(String[] args) {
    double radius = 5.0;  // Circle radius
    double area = calculateCircleArea(radius);

    // Print area with two decimal places
    System.out.printf("Area: %.2f%n", area);
}

File: src/data/snippets/functions/snippet.c
#include <stdio.h>
#include <math.h>

// Function to calculate the area of a circle
double calculate_circle_area(double radius) {
    // Area formula using pow() for exponentiation
    return M_PI * pow(radius, 2); 
}

int main() {
    double radius = 5.0;  // Circle radius
    double area = calculate_circle_area(radius);

    // Print area with two decimal places
    printf("Area: %.2f\n", area);
    return 0;
}
